wrappers/python/simtk/openmm/app/gromacstopfile.py

```diff
@@ -857,19 +880,44 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
                         continue # We'll use the automatically generated parameters
                     atom1params = nb.getParticleParameters(baseAtomIndex+atoms[0])
                     atom2params = nb.getParticleParameters(baseAtomIndex+atoms[1])
-                    exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atoms[1], atom1params[0]*atom2params[0]*fudgeQQ, params[0], params[1]))
+                    # enable tracking pairs and exceptions separately
+                    if self._defaults[1] == '1':
+                        pairs.append((baseAtomIndex+atoms[0], baseAtomIndex+atoms[1], atom1params[0]*atom2params[0]*fudgeQQ, params[0], params[1]))
+                    elif self._defaults[1] == '2':
+                        exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atoms[1], atom1params[0]*atom2params[0]*fudgeQQ, params[0], params[1]))
                 for fields in moleculeType.exclusions:
                     atoms = [int(x)-1 for x in fields]
                     for atom in atoms[1:]:
                         if atom > atoms[0]:
-                            exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atom, 0, 0, 0))
+                            if self._defaults[1] == '1':
+                                exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atom))
+                            elif self._defaults[1] == '2':
+                                exceptions.append((baseAtomIndex+atoms[0], baseAtomIndex+atom, 0, 0, 0))
 
 
         # Create nonbonded exceptions.
+        if self._defaults[1] == '1':
+            # implement named exceptions (generally used for 1-4 interactions)
+            pair_bond = mm.CustomBondForce('138.935456*q/r-C/r^6+A/r^12')
```

Comment:
If you use NonbondedForce for the electrostatics you can also have it handle the exception parameters (rather than requiring that they be done with a CustomBondForce).